packages/devextreme/testing/tests/DevExpress.ui.widgets.dataGrid/keyboardNavigation.accessibility.tests.js

```diff
@@ -906,4 +906,74 @@ QUnit.module('Keyboard navigation accessibility', {
         assert.ok($headerItem.is(':focus'), 'Header cell has focus');
         assert.notOk($headersElement.hasClass('dx-state-focused'), 'Headers main element has no dx-state-focused class');
     });
+
+    // T1216832
+    testInDesktop('Navigation should not get stuck in the grid when the first cell is fixed and a template with links is specified for it', function(assert) {
```

Comment:
Let's split this test case into two:
- The first one will check the ```navigationOfAllCells```
- The second one will check the ```inert``` attribute

And let's add the third test case like in the original ticket issue:
1) Focus on the last cell (in the DataGrid with fixed columns)
2) Press the ```Tab```
3) Check that DataGrid loses focus or that the element after the DataGrid gains focus

We will check the original issue case with this test case
(In this issue focus goes to the fixed column instead of "move out" from the DataGrid)